web/src/routes/admin/library-management/+page.svelte

```diff
@@ -397,6 +410,12 @@
                     <Portal target="body">
                       <ContextMenu {...contextMenuPosition} on:outclick={() => onMenuExit()}>
                         <MenuOption on:click={() => onRenameClicked()} text={`Rename`} />
+                        <MenuOption
+                          on:click={() => onToggleReadOnlyClicked()}
+                          text={(selectedLibrary?.isReadOnly ||
```

Comment:
This should not be nullable

Reply:
we could probably change the `{#if showContextMenu}` above to `{#if showContextMenu && selectedLibrary}` to simplify all the accesses to `selectedLibrary` within